res/controllers/common-controller-scripts.js

```diff
@@ -268,13 +268,13 @@ script.absoluteNonLinInverse = function(value, low, mid, high, min, max) {
     if (!max) {
         max = 127
     }
-    var center = (max - min) / 2
-    var result
+    var center = (max-min)/2;
+    var result;
 
-    if (value == mid) {
-        return center
-    } else if (value < mid) {
-        result = (center / (mid - low)) * (value - low)
+    if (value==mid) {
+        return center;
+    } else if (value<mid) {
+        result = (center/(mid-low)) * (value-low);
```

Comment:
Can you restore the operator spacing in the changed lines above? This is inconsistent with the spacing used below.